moon-openshift.yaml

```diff
@@ -49,6 +49,15 @@ objects:
      spec:
        selector:
          moon: browser
+       ports:
```

Comment:
As mentioned below this is only needed for old Kubernetes versions. So this change is not needed.

Reply:
Nope. It's needed for OKD 3.11. It's actual and last version of OKD in 3.x family. OKD 4.x is not a product ready.